src/core/execute.c

```diff
@@ -5248,9 +5248,12 @@ int exec_spawn(Unit *unit,
                         if (r < 0)
                                 return log_unit_error_errno(unit, r, "Failed to create control group '%s': %m", subcgroup_path);
 
-                        /* Normally we would not propagate the oomd xattrs to children but since we created this
+                        /* Normally we would not propagate the xattrs to children but since we created this
                          * sub-cgroup internally we should do it. */
                         cgroup_oomd_xattr_apply(unit, subcgroup_path);
+                        r = cgroup_log_xattr_apply(unit, subcgroup_path);
+                        if (r < 0)
+                                log_unit_debug_errno(unit, r, "Failed to set user.journald_log_filter_patterns cgroup xattr, ignoring: %m");
```

Comment:
Hmm, having one of the two calls print a message and not the other looks strange.
And afaict, cgroup_log_xattr_apply() would already log because it calls unit_set_xattr_graceful() which logs.
Would it be possible to move the logging fully into cgroup_log_xattr_apply() so that we don't get duplicated logging?